Logs/AssetImportWorker0.log

```diff
@@ -0,0 +1,108 @@
+Using pre-set license
```

Comment:
these log files can actually be omitted from git entirely. you can add something called a `.gitignore` to the repo and it makes sure git doesn't track changes that don't need to be there. i'll send you an example in slack.